### PR TITLE
[vk] Catch author blocked error message

### PR DESCRIPTION
**Pull request by bugchecker**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Some videos blocked following its author. Example link (video in blocked group):
https://vk.com/search?c%5Bq%5D=%D0%9F%D1%80%D1%8B%D0%B6%D0%BE%D0%BA%20c%20%D0%BA%D1%80%D0%B0%D0%BD%D0%B0%20%D0%B2%20%D1%81%D1%82%D0%B8%D0%BB%D0%B5%20%D0%A7%D0%B5%D0%BB%D0%BE%D0%B2%D0%B5%D0%BA%D0%B0-%D0%BF%D0%B0%D1%83%D0%BA%D0%B0&c%5Bsection%5D=video&c%5Bsort%5D=2&z=video-10639516_456240611

### CODE REVIEW

File: youtube_dl/extractor/vk.py [hunk 1]
```diff
@@ -328,6 +328,12 @@ def _real_extract(self, url):
 
             r'<!>Access denied':
             'Access denied to video %s.',
+
+            r'<!>Видеозапись недоступна, так как её автор был заблокирован.':
+            'Author of video %s has been blocked.',
+
+            r'<!>This video is no longer available, because its author has been blocked.':
+            'Author of video %s has been blocked.',
```
Comment (dstftw) [inline youtube_dl/extractor/vk.py]: Add a tests.

Comment (bugchecker) [inline youtube_dl/extractor/vk.py]: Using `only_matching` is it correct?
```
$ python -m youtube_dl test:vk_16
[TestURL] Test URL: https://vk.com/search?c%5Bq%5D=%D0%9F%D1%80%D1%8B%D0%B6%D0%BE%D0%BA%20c%20%D0%BA%D1%80%D0%B0%D0%BD%D0%B0%20%D0%B2%20%D1%81%D1%82%D0%B8%D0%BB%D0%B5%20%D0%A7%D0%B5%D0%BB%D0%BE%D0%B2%D0%B5%D0%BA%D0%B0-%D0%BF%D0%B0%D1%83%D0%BA%D0%B0&c%5Bsection%5D=video&c%5Bsort%5D=2&z=video-10639516_456240611
[vk] -10639516_456240611: Downloading webpage
ERROR: Author of video -10639516_456240611 has been blocked.
```

**Comment by dstftw** [comment]
Cherry-picked, thanks.

**Comment by bugchecker** [comment]
@dstftw test link works only for Russia. 
May be add: `'skip': 'Only works from Russia',`

**Comment by dstftw** [comment]
`only_matching` tests are only matched against `_VALID_URL`.

**Pull request closed without merging**